sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/implementation/models/CallParticipantInternal.java

```diff
@@ -61,4 +67,24 @@ public CallParticipantInternal setIsMuted(Boolean isMuted) {
         this.isMuted = isMuted;
         return this;
     }
+
+    /**
+     * Get the isOnHold property: Is participant on hold.
+     *
+     * @return the isOnHold value.
+     */
+    public Boolean isOnHold() {
```

Comment:
Please update `CallParticipant` as well: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/CallParticipant.java